src/si/momentum.rs

```diff
@@ -0,0 +1,201 @@
+//! Momentum (base unit kilogram meter per second, kg<sup>1</sup> · m<sup>1</sup> · s<sup>-1</sup>).
+
+quantity! {
+    /// Momentum (base unit kilogram meter per second, kg<sup>1</sup> · m<sup>1</sup> · s<sup>-1</sup>).
+    quantity: Momentum; "momentum";
+    /// Momentum dimension, kg<sup>1</sup> · m<sup>1</sup> · s<sup>-1</sup>).
+    dimension: ISQ<
+        P1,     // length
+        P1,     // mass
+        N1,     // time
+        Z0,     // electric current
+        Z0,     // thermodynamic temperature
+        Z0,     // amount of substance
+        Z0>;    // luminous intensity
+    units {
+        @yottagram_meter_per_second: prefix!(yotta) / prefix!(kilo); "Yg · m/s", "yottagram meter per second",
```

Comment:
Wrap at 100 characters, just before the singular description.

Reply:
done